internal/pkg/service/common/task/node.go

```diff
@@ -319,6 +320,23 @@ func (n *Node) runTask(logger log.Logger, task Task, cfg Config) (result Result,
 		task.Result = result.Result
 	} else {
 		task.Error = result.Error.Error()
+		task.UserError = &Error{}
+
+		if errWithName, ok := result.Error.(svcerrors.WithName); ok {
+			task.UserError.Name = errWithName.ErrorName()
+		} else {
+			task.UserError.Name = "unknownError"
+		}
+
+		if errWithUserMessage, ok := result.Error.(svcerrors.WithUserMessage); ok {
+			task.UserError.Message = errWithUserMessage.ErrorUserMessage()
+		} else {
+			task.UserError.Message = "Unknown error"
+		}
+
+		if errWithExceptionID, ok := result.Error.(svcerrors.WithExceptionID); ok {
+			task.UserError.ExceptionID = errWithExceptionID.ErrorExceptionID()
+		}
 	}
```

Comment:
The exceptionID should be always present in the message + optionally in the `expcetionId` field.

Workflow:
- User gets (in the UI) `something wrong `message with exception ID.
- He contacts our support.
- We see in the logs the full error message, with the same exception ID.

Reply:
Please see the `httpserver` package as reference, if there is no exceptionID you should generate it, for the log purposes mentioned above. The key is to somehow implement the `workflow`.